.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-with-various-java-version:
+    strategy:
+      matrix:
+        version: [8, 11, 17, 21]
+        project:
+            - 'microservice-transaction-sample'
+            - 'multi-storage-transaction-sample'
+            - 'scalardb-kotlin-sample'
+            - 'scalardb-sample'
+            - 'scalardb-sql-jdbc-sample'
+            - 'spring-data-microservice-transaction-sample'
+            - 'spring-data-multi-storage-transaction-sample'
+            - 'spring-data-sample'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK ${{ matrix.version }}
+      uses: actions/setup-java@v4
+      with:
+        java-version: ${{ matrix.version }}
+        distribution: 'temurin'
+
+    - name: Setup Gradle
```

```suggestion
    - name: Set up Gradle
```
It's a very minor thing, but in the previous step, it uses `Set up` instead of `Setup`. So, it might be better to use the same term `Set up` here.

---

@kota2and3kan Those names are generated automatically when using `Java with Gradle` workflow. So, I left them. Do you think we should still fix it in that case?

---

Ah, I see. I didn't know that.
In that case, I think there is no problem.
So, let's keep it as is!